xgboost_synthetic/build-train-deploy.ipynb

```diff
@@ -29,6 +29,126 @@
     "Import the libraries required to train this model."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
```

Comment:
I don't think this is necessary per comment above. When you launch the notebook you should select "add gcp credential" and this should set GOOOGLE_APPLICATION_CREDENTIALS automatically